wallet/src/cli/programs/token.rs

```diff
@@ -386,6 +669,34 @@ impl WalletSubcommand for TokenProgramSubcommandPublic {
                     .await?;
                 Ok(SubcommandReturnValue::Empty)
             }
+            TokenProgramSubcommandPublic::BurnToken {
+                definition_account_id,
+                holder_account_id,
+                amount,
+            } => {
+                Token(wallet_core)
+                    .send_burn_transaction(
+                        definition_account_id.parse().unwrap(),
```

Comment:
So we're panicking if user provided wrong ids... It should be an error then, not a panic. Let's have an issue for that?